#### DatasetS1-phylo_host_country_genome_features.xlsx
This dataset provides the EzClermont grouping, country, and host information for 2,661 wild-type E. coli strains used in this study. This dataset is plotted in Fig. S1a-b and is used as the color scheme for Fig. S1c-f. More information is provided in the file.
#### DatasetS2-Ecoli_genomes_and_allele_ids.xlsx.
This dataset includes a matrix (2661 strains x 4349 genes) that describes the specific nucleic acid allele id found in each strain for each open reading frame. It is the result of running the bidirectional best blast tool from RAST for 2661 strains to identify ORFs and their corresponding Blattner numbers in wild-type strains. This data is plotted in Fig. S1c and used in Fig S1d-f. More information is provided in the file.
#### DatasetS3-allele_sequences.zip.
This dataset describes the codon sequence, amino acid sequence, strain count, and QCQA decision for each allele found across 2661 strains described in Dataset S2. This data is plotted in Fig. S2 and used in Fig. S1d-f. More information is provided in the file. 
#### DatasetS4-The_WT_alleleome_1_3M_AAs.xlsx.
This dataset is the E. coli wild-type alleleome and describes natural sequence variation found across 1.3 million codon positions. The number of strains carrying the dominant codon, dominant AA, variant codons, and variant AAs, are provided for each position. This data is plotted in Fig. 2, Fig. S3 and Fig. S7. More information is provided in the file.
#### DatasetS5-ALE_and_LTEE_mutations.xls.
This dataset contains the QCQA’d (see Fig. S5) mutations from ALEdb (private and public) and the LTEE database. For each mutation, the WT occurrence of the mutation’s position (in DatasetS4) is calculated. This data is plotted in Fig. 4-5 and Fig. S6. More information provided in the file.
#### DatasetS6-Mutations_in_WT_and_Lab_strains_.xlsx
This dataset groups the mutations described in the WT (Dataset S4, variants) and laboratory (DatasetS5) strains by their specific initial-final codon change (Codon1Codon2).  This dataset provides a total count of each observed mutation across the datasets, a Grantham score associated with each mutation, and the number of expected mutations calculated by our model (Fig. S4) for the WT strains. This data is plotted in Fig. 3, Fig. 6, and Fig. S7. More information is provided in the file.
#### DatasetS7-stat_tables_sfig8-10.xlsx
This dataset provides the p-values for the statistical tests used in this study and is plotted in Fig. S8-10. This dataset reflects the statistical analysis results of DatasetS6, for mutations grouped by initial codon or initial amino acid. More information is provided in the file. 
